spec/issues/range_parameter_delimited_by_dash_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+describe '#XXX nested entity given as string' do
```

Comment:
Sorry for the naming, my first PR ever ... should I open an issue for this, to get the number?